schema.sql

```diff
@@ -6,16 +6,16 @@ DROP TABLE IF EXISTS product;
 
```

Comment:
For line 3 the postgress command for use  is  \c database name. i think also postgres only supports if exists, so the command on line on would have to be split to DROP DATABASE IF EXISTS and then the CREATE DATABASE